src/app/content/highlights/components/utils/showConfirmation.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { RawIntlProvider } from 'react-intl';
 import uuid from 'uuid/v4';
+import createIntl from '../../../../messages/createIntl';
 import { AppServices } from '../../../../types';
 import { assertDocument, assertNotNull } from '../../../../utils';
 import ConfirmationModal from '../ConfirmationModal';
 
 export default async(services: AppServices) => {
   const document = assertDocument();
   const domNode = document.createElement('div');
+  const intl = await createIntl().getIntlObject('en');
```

Comment:
`ConfirmationModal` will always be in English?

Reply:
just a placeholder, sorry. will ping you once the `showConfirmation` logic is ready.